qiskit_experiments/base_analysis.py

```diff
@@ -90,3 +92,34 @@ def _run_analysis(self, experiment_data, **options):
                    None, a single figure, or a list of figures.
         """
         pass
+
+    @staticmethod
+    def exp_fit_fun(x, a, tau, c):
```

Comment:
Since these are staticmethods there isn't really any reason to add make them part of the base class, they are just functions that can be imported from library when they are needed.

I wouls suggest moving them to an `analysis_functions.py` file for now, and later working on making `CurveFitAnalysis` and `ExpFitAnalysis` classes later that can be reused by experiments that have all there analysis as curve fitting (eg this and RB)

Reply:
The idea is to signal that these functions are related to analysis. Anyway, this is not important, because since it's all temporary and will go away after #19. I moved the functions to a separate file.